src/Persistence/PersistentObjectFactory.php

```diff
@@ -301,10 +310,6 @@ final protected function normalizeAttribute(mixed $value, string $name): mixed
             return $value->isPersisted() ? $value->refresh()->object() : $value->object();
         }
 
-        if (\is_callable($value)) {
-            $value = $value();
-        }
```

Comment:
This was coming from a misunderstanding from my side... I've always thought we were allowed to do something like this:
```php
class SomeFactory extends ModelFactory
{
    protected function getDefaults(): array
    {
		return [
			'field' => fn() => FieldFacotry::findOrCreate()
        ];
	}
}
```
but it actually introduce a bug if the value passed is invokable (I had the case in one of the project where I tested this version)